server/handlers/rawCaptureHandler.js

```diff
@@ -8,8 +8,6 @@ const {
 } = require('../utils/helper');
 
 const rawCaptureGetQuerySchema = Joi.object({
-  offset: Joi.number().integer().greater(-1),
-  limit: Joi.number().integer().greater(0),
```

Comment:
Is there a particular reason why limit and offset were removed?

Reply:
My bad.  I might've mistook which project I had open.